### PR TITLE
Improvements to test vector generation

### PR DESCRIPTION
**Pull request by junyechen1996**
Add an environmental variable TEST_VECTOR to control test vector generation, and an optional environmental variable TEST_VECTOR_PATH to specify a custom path where test vectors are generated. Add a new target in Makefile to generate all test vectors in VDAF spec.

### CODE REVIEW

File: poc/Makefile [hunk 1]
```diff
@@ -10,3 +10,9 @@ test:
 	sage -python vdaf.py
 	sage -python vdaf_prio3.py
 	sage -python vdaf_poplar1.py
+
+generate_test_vectors:
+	TEST_VECTOR=TRUE sage -python idpf_poplar.py
+	TEST_VECTOR=TRUE sage -python vdaf_prio3.py
+	TEST_VECTOR=TRUE sage -python vdaf_poplar1.py
+	TEST_VECTOR=TRUE sage -python xof.py
```
Comment (cjpatton) [inline poc/Makefile:18]: This list of commands needs to be kept in sync with the files for which we generate test vectors.

Luckily, I don't think this is necessary. To set the `TEST_VECTOR` variable to `true`, just do:
```
make TEST_VECTOR=true
```

```suggestion
```

File: poc/README.md [hunk 2]
```diff
@@ -19,10 +19,26 @@ Version 3.20.0 or later is required.
 
 ## Generating test vectors
```
Comment (cjpatton) [inline poc/README.md:21]: Remove `generate_test_vectors` from the Makefile (see above) and update these instructions.

**Review by cjpatton — COMMENTED** [review]
Great idea! One minor suggestion.

File: poc/README.md (hunk 2 above)
Comment (junyechen1996) [reply poc/README.md:21]: Done.

File: poc/Makefile (hunk 1 above)
Comment (junyechen1996) [reply poc/Makefile]: Good point. Done.

**Review by cjpatton — APPROVED** [review]
Please squash and then I'll merge.

**Comment by junyechen1996** [comment]
Squashed. Should be good to merge after CI.

**Review by divergentdave — APPROVED** [review]

**Pull request merged**